facefusion/face_classifier.py

```diff
@@ -4,35 +4,69 @@
 
 from facefusion import inference_manager
 from facefusion.download import conditional_download_hashes, conditional_download_sources
-from facefusion.face_helper import warp_face_by_translation
+from facefusion.face_helper import warp_face_by_face_landmark_5
 from facefusion.filesystem import resolve_relative_path
 from facefusion.thread_helper import conditional_thread_semaphore
-from facefusion.typing import BoundingBox, InferencePool, ModelOptions, ModelSet, Prediction, VisionFrame
+from facefusion.typing import FaceLandmark5, InferencePool, ModelOptions, ModelSet, Prediction, VisionFrame
 
 MODEL_SET : ModelSet =\
 {
-	'gender_age':
+	'fair_face':
 	{
 		'hashes':
 		{
 			'gender_age':
 			{
-				'url': 'https://github.com/facefusion/facefusion-assets/releases/download/models-3.0.0/gender_age.hash',
-				'path': resolve_relative_path('../.assets/models/gender_age.hash')
+				'url': 'https://huggingface.co/bluefoxcreation/gender_age/resolve/main/fair_face.hash',
+				'path': resolve_relative_path('../.assets/models/fair_face.hash')
 			}
 		},
 		'sources':
 		{
 			'gender_age':
 			{
-				'url': 'https://github.com/facefusion/facefusion-assets/releases/download/models-3.0.0/gender_age.onnx',
-				'path': resolve_relative_path('../.assets/models/gender_age.onnx')
+				'url': 'https://huggingface.co/bluefoxcreation/gender_age/resolve/main/fair_face.onnx',
+				'path': resolve_relative_path('../.assets/models/fair_face.onnx')
 			}
-		}
+		},
+		'template': 'arcface_112_v2',
+		'size': (224, 224),
+		'mean': [ 0.485, 0.456, 0.406 ],
+		'standard_deviation': [ 0.229, 0.224, 0.225 ]
 	}
 }
 
 
+def categorize_age(age_id : int) -> str:
```

Comment:
the categories have been introduced as the gender age is not accurate enough.

we can either switch to 0-10 ranges or introduce more categories